Modules/Server/Tests/PocketCastsServerTests/ServerHelperTests.swift

```diff
@@ -0,0 +1,99 @@
+        XCTAssertEqual(firstPodcastUpdates?[1].episodeDescription, nil)
+        XCTAssertEqual(firstPodcastUpdates?[1].detailedDescription, nil)
+        XCTAssertEqual(firstPodcastUpdates?[1].fileType, "audio/mp3")
+        XCTAssertEqual(firstPodcastUpdates?[1].sizeInBytes, 2271)
+        XCTAssertEqual(firstPodcastUpdates?[1].duration, 2271)
+        XCTAssertEqual(firstPodcastUpdates?[1].episodeType, "full")
+        XCTAssertEqual(firstPodcastUpdates?[1].seasonNumber, 0)
+        XCTAssertEqual(firstPodcastUpdates?[1].episodeNumber, 0)
+        XCTAssertEqual(firstPodcastUpdates?[1].publishedDate, "2024-01-26 13:32:42")
+
+        XCTAssertEqual(secondPodcastUpdates?[0].uuid, "161160ef-fc5b-4165-b2a3-c52963733f0c")
+        XCTAssertEqual(secondPodcastUpdates?[0].url, "https://sphinx.acast.com/p/open/s/593eded1acfa040562f3480b/e/65ae84931a5c7e0017941cbc/media.mp3")
+        XCTAssertEqual(secondPodcastUpdates?[0].title, "Episode 877")
+        XCTAssertEqual(secondPodcastUpdates?[0].episodeDescription, nil)
+        XCTAssertEqual(secondPodcastUpdates?[0].detailedDescription, nil)
+        XCTAssertEqual(secondPodcastUpdates?[0].fileType, "audio/mp3")
+        XCTAssertEqual(secondPodcastUpdates?[0].sizeInBytes, 89228708)
+        XCTAssertEqual(secondPodcastUpdates?[0].duration, 3605)
+        XCTAssertEqual(secondPodcastUpdates?[0].episodeType, "full")
+        XCTAssertEqual(secondPodcastUpdates?[0].seasonNumber, 0)
+        XCTAssertEqual(secondPodcastUpdates?[0].episodeNumber, 0)
+        XCTAssertEqual(secondPodcastUpdates?[0].publishedDate, "2024-01-26 15:00:52")
+    }
+}
+
+public class JSONHelper {
+    public static func loadJSON(
+        withFile fileName: String,
+        inBundleWithName bundle: String = "PocketCastsServer",
+        subdirectory: String = "Fixtures") -> Data? {
+        let bundle = Bundle.currentModule(name: bundle)
+
+        if let url = bundle.url(forResource: fileName, withExtension: "json", subdirectory: subdirectory) {
+            do {
+                let data = try Data(contentsOf: url)
+                return data
+            } catch {
+                debugPrint(error)
+            }
+        } else {
+            debugPrint("Could not find the json file: \(fileName) in bundle: \(bundle)")
+        }
+        return nil
+    }
+}
+
+extension Foundation.Bundle {
+    static func currentModule(name: String) -> Bundle {
+        var thisModuleName = name + "_" + name
+        var url = Bundle.main.bundleURL
+
+        for bundle in Bundle.allBundles where bundle.bundlePath.hasSuffix(".xctest") {
+            url = bundle.bundleURL.deletingLastPathComponent()
+            thisModuleName = thisModuleName.appending("Tests")
+        }
+
+        url = url.appendingPathComponent("\(thisModuleName).bundle")
+
+        guard let bundle = Bundle(url: url) else {
+            fatalError("Foundation.Bundle.module could not load resource bundle: \(url.path)")
+        }
+
+        return bundle
+    }
+}
```

Comment:
We can use `Bundle.module` to get the resource path for the tests, and remove the extra code here:

```suggestion
public class JSONHelper {
    public static func loadJSON(withFile fileName: String, subdirectory: String = "Fixtures") -> Data? {
        let bundle = Bundle.module

        if let url = bundle.url(forResource: fileName, withExtension: "json", subdirectory: subdirectory) {
            do {
                return try Data(contentsOf: url)
            } catch {
                debugPrint(error)
            }
        } else {
            debugPrint("Could not find the json file: \(fileName) in bundle: \(bundle)")
        }
        return nil
    }
}
```